cpp/models/abm/testing_strategy.h

```diff
@@ -146,7 +150,6 @@ class TestingScheme
 
 private:
     TestingCriteria m_testing_criteria; ///< TestingCriteria of the scheme.
-    TimeSpan m_minimal_time_since_last_test; ///< Shortest period of time between two tests.
```

Comment:
I think this should be kept, but with a different name. See my other comments in the parameters.